### DISCUSSION TITLE
DENS / urfan.web.id host

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#97] [Show and tell]
NOTE : gathered using my usual verification program + VLC for answer testing. Links being here are the ones available everywhere.

.35 = DENS host
.37 = DENS w/out geographical restrictions.

Session qwr9ew

1. TransTV / http://210.210.155.35/qwr9ew/s/s05/index.m3u8
2. SCTV / http://210.210.155.35/qwr9ew/s/s03/index.m3u8
3. NET. / http://210.210.155.35/qwr9ew/s/s08/index.m3u8
4. Elshinta TV / http://210.210.155.35/qwr9ew/s/s10/index.m3u8
5. TVRI / http://210.210.155.35/qwr9ew/s/s11/index.m3u8
6. DAAI TV / http://210.210.155.35/qwr9ew/s/s13/index.m3u8
7. Rodja TV / http://210.210.155.35/qwr9ew/s/s14/index.m3u8
8. Al Jazeera Arabic / http://210.210.155.35/qwr9ew/s/s16/index.m3u8
9. Al Jazeera English / http://210.210.155.35/qwr9ew/s/s17/index.m3u8
10. CCTV 4 / http://210.210.155.35/qwr9ew/s/s19/index.m3u8
11. http://210.210.155.35/qwr9ew/s/s20/index.m3u8
12. Phoenix InfoNews / http://210.210.155.35/qwr9ew/s/s21/index.m3u8
13. France 24 / http://210.210.155.35/qwr9ew/s/s22/index.m3u8
14. http://210.210.155.35/qwr9ew/s/s25/index.m3u8
15. Saudi Sunnah / http://210.210.155.35/qwr9ew/s/s27/index.m3u8
16. Saudi Quran / http://210.210.155.35/qwr9ew/s/s28/index.m3u8
17. Celestial Classic Movies (Indonesian Sub) / http://210.210.155.35/qwr9ew/s/s31/index.m3u8
18. Zee Bioskop / http://210.210.155.35/qwr9ew/s/s32/index.m3u8
19. THRILL TV (Indonesian Sub) / http://210.210.155.35/qwr9ew/s/s34/index.m3u8
20. RTP 1 / http://210.210.155.35/qwr9ew/s/s38/index.m3u8
21. TBN Asia / http://210.210.155.35/qwr9ew/s/s39/index.m3u8
22. O Channel HD / http://210.210.155.35/qwr9ew/s/s41/index.m3u8
23. http://210.210.155.35/qwr9ew/s/s44/index.m3u8
24. http://210.210.155.35/qwr9ew/s/s45/index.m3u8
25. Chinese channel, however idk what it is, screencap below / http://210.210.155.35/qwr9ew/s/s50/index.m3u8

![vlcsnap-2022-03-26-12h42m33s679](https://user-images.githubusercontent.com/30985701/160237878-83a70bb0-7d8c-4d8a-8584-5f72bc2bbcaf.png)

Session uq2663 (playlist.urfan.web.id)

27. K+ (Indonesian Sub) / http://210.210.155.35/uq2663/h/h08/index.m3u8
28. MotorVision / http://210.210.155.35/uq2663/h/h09/index.m3u8
29. Trans 7 HD / http://210.210.155.35/uq2663/h/h11/index.m3u8
30. Gasproll / http://210.210.155.35/uq2663/h/h12/index.m3u8
31. My Cinema Europe Asia / http://210.210.155.35/uq2663/h/h18/index.m3u8
32. GEM / http://210.210.155.35/uq2663/h/h19/index.m3u8
33. EGG Network / http://210.210.155.35/uq2663/h/h22/index.m3u8
34. NHK World Japan / http://210.210.155.35/uq2663/h/h23/index.m3u8
35. http://210.210.155.35/uq2663/h/h28/index.m3u8 (cut)
36. CNA / http://210.210.155.35/uq2663/h/h29/index.m3u8
37. NTV HD / http://210.210.155.35/uq2663/h/h37/index.m3u8
38. TVone HD / http://210.210.155.35/uq2663/h/h40/index.m3u8
39. http://210.210.155.35/uq2663/h/h94/index.m3u8 (cut)
40. http://210.210.155.35/uq2663/h/h95/index.m3u8 (cut)
41. http://210.210.155.35/uq2663/h/h141/index.m3u8 (cut)
42. http://210.210.155.35/uq2663/h/h142/index.m3u8 (cut)
43. One / http://210.210.155.35/uq2663/h/h143/index.m3u8 (cut)
44. http://210.210.155.35/uq2663/h/h150/index.m3u8 (cut)

Session dr9445 (playlist.urfan.web.id)

45. TV5 Monde Style HD (french channel) http://210.210.155.35/dr9445/h/h01/index.m3u8
46. Aniplus Asia http://210.210.155.35/dr9445/h/h02/index.m3u8
47. Bloomberg / http://210.210.155.35/dr9445/h/h03/index.m3u8
48. CinemaWorld Asia / http://210.210.155.35/dr9445/h/h04/index.m3u8
49. Fight Channel / http://210.210.155.35/dr9445/h/h05/index.m3u8
50. Kix Asia / http://210.210.155.35/dr9445/h/h07/index.m3u8
51. http://210.210.155.35/dr9445/h/h09/index.m3u8
52. RTV / http://210.210.155.35/dr9445/h/h10/index.m3u8
53. http://210.210.155.35/dr9445/h/h11/index.m3u8
54. Celestial Movies Asia / http://210.210.155.35/dr9445/h/h14/index.m3u8
55. Rock Extreme Asia / http://210.210.155.35/dr9445/h/h15/index.m3u8
56. Rock Entertainment Asia / http://210.210.155.35/dr9445/h/h16/index.m3u8
57. TVN / http://210.210.155.35/dr9445/h/h20/index.m3u8
58. TVN Movies / http://210.210.155.35/dr9445/h/h21/index.m3u8
59. Hits / http://210.210.155.35/dr9445/h/h37/index.m3u8
60. AXN East Asia / http://210.210.155.35/dr9445/h/h141/index.m3u8
61. Animax Asia / http://210.210.155.35/dr9445/h/h144/index.m3u8

.36 = reserved CBN IP

1. DENS Live streaming TV 1 : http://210.210.155.36/s/s100/live.m3u8

**Comment by F640** [comment]
Actual host links:
uq2663 = rr.dens.tv
dr9445 = rr1.dens.tv
qwr9ew = rr2.dens.tv
x6bnqe = rr3.dens.tv
is6knm = rrelshinta.dens.tv

Replacing 210.210.155.35 with 210.210.155.37 will remove geographical restriction to some country for most Dens TV links.
